Formula/k/kyua.rb

```diff
@@ -0,0 +1,39 @@
+class Kyua < Formula
+  desc "Testing framework for infrastructure software"
+  homepage "https://github.com/freebsd/kyua"
+  url "https://github.com/freebsd/kyua/releases/download/kyua-0.14.1/kyua-0.14.1.tar.gz"
+  sha256 "3caf30a7e316f4f21c32e1c419ec80371fe113e3eed10ba1db9e6efc7ee15ecb"
+  license "BSD-3-Clause"
+  head "https://github.com/freebsd/kyua.git", branch: "master"
+
+  # Upstream creates pre-released tags as part of their release process, so use
+  # the `GithubLatest` strategy to pick up the latest non-release version prior
+  # to the official release.
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
+
+  depends_on "pkgconf" => :build
+  depends_on "atf"
+  depends_on "lua"
+  depends_on "lutok"
+
+  uses_from_macos "sqlite"
+
+  def install
+    ENV.append "CPPFLAGS", "-I#{Formula["lua"].opt_include}/lua"
+
+    system "./configure", "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--enable-atf",
+                          "--prefix=#{prefix}"
```

Comment:
We have standard options here, can you check?
- https://docs.brew.sh/Formula-Cookbook#std_configure_args